### PR TITLE
[CN-2206] /cdk sample connectors repo enum labels

### PR DESCRIPTION
**Pull request by Elliot-Ball-Tray**
This sample connector has 1 operation which has inputs that cover examples of:
field titles, descriptions (i.e. tooltips), default values, advanced properties, enums (i.e. static dropdowns) and oneOf (i.e. dropdowns).

### CODE REVIEW

**Review by Samzy — APPROVED** [review]
Looks good, would be nice to format the JSON so it's easier to read and add the new line errors to files that require it.

File: property-panel-ux/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (gnyani-trayio) [inline property-panel-ux/tsconfig.json:1]: should we generate these examples with a ts config that fixes the issues with vs-code imports?

File: property-panel-ux/connector.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+    "name": "property-panel-ux",
+    "version": "1.0",
+    "title": "Property Panel UX",
+    "description": "An example connector showing how to use UI features in the property panel, including: field titles, descriptions (i.e. tooltips), default values, advanced properties, enums (i.e. static dropdowns) and oneOf (i.e. dropdowns).",
+    "service": {
+        "name": "json-placeholder",
+        "version": "1"
+    },
```
Comment (joaoTray) [inline property-panel-ux/connector.json:9]: what is this field?
needs to be changed as well?

File: property-panel-ux/src/create_post/input.json [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+            "type": "string"
+        },
+        "userId": {
+            "title": "User ID",
+            "description": "The ID of the user to create the post for",
+            "type": "number"
+        },
+        "owner": {
+            "title": "Owner",
+            "description": "The owner of the post (optional, either a non profit or business)",
```
Comment (joaoTray) [inline property-panel-ux/src/create_post/input.json:35]: is this correct? 
If is oneOf, should be `required`, no?

File: README.md [hunk 4]
```diff
@@ -3,10 +3,11 @@
...
    - `generate_sha_256_hash` is a composite operation that demonstrates a more complex use case. It uses the `crypto` library to generate a SHA-256 hash from a given string.
    - `create_or_update_post` is a complex composite operation that demonstrates a use case that conditionally makes a POST or PUT request based on the existence of a post with a given ID. If the post `id` exists as an input property, the operation makes a PUT request to update the post. If the post `id` does not exist as an input property, the operation makes a POST request to create a new post.
 3. ddl-example: this connector includes 3 operations that demonstrate how to create and use a authenticated ddl.
 ([Documentation for DDL Implementation](https://developer.tray.io/developer-portal/cdk/dsl-reference/#ddl-dynamic-dropdown-lists)).
+4. property-panel-ux: An example connector showing how to use UI features in the property panel, including: field titles, descriptions (i.e. tooltips), default values, advanced properties, enums (i.e. static dropdowns) and oneOf (i.e. dropdowns).
```
Comment (joaoTray) [inline README.md:12]: not saying that we should do it, but just as a suggestion: Would be good to have in the README an image of the property panel.

**Review by joaoTray — APPROVED** [review]

**Pull request merged**